common/lib/util/eventemitter.js

```diff
@@ -57,7 +57,7 @@ var EventEmitter = (function() {
 			/* ... or we take event to be the actual event name and listener to be all */
 		}
 		var listeners, idx = -1;
-		if(event === null) {
+		if((event === null) || (event === undefined)) {
```

Comment:
I wasn't asking for the code, I was wondering if a similar method existed elsewhere to reduce code duplication!